app/frontend/private/pos/orders/styles.css

```diff
@@ -0,0 +1,32 @@
+* {
+    font-family: sans-serif;
+}
+
+.header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding: 12px;
+    height: 80px;
+    background-color: #111;
+}
+
+.header-title {
+    color: #eee;
+    margin: auto 0;
+    font-weight: 700 !important;
+}
+
+.header-buttons {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    width: auto;
+}
+
+.header-button {
+    margin-left: 16px;
+    font-size: 24px;
+    font-weight: 700;
+}
```

Comment:
We ending files without whitespace Jason? 🤨

Reply:
???